src/components/SwipeButton/index.js

```diff
@@ -214,6 +218,8 @@ SwipeButton.propTypes = {
   titleColor: PropTypes.string,
   titleFontSize: PropTypes.number,
   width: PropTypes.number,
+  shouldResetAfterSuccess: PropTypes.bool,
+  resetAfterSuccessDuration: PropTypes.number,
```

Comment:
How about renaming `resetAfterSuccessDuration` to `resetAfterSuccessAnimDuration`

```suggestion
  resetAfterSuccessAnimDuration: PropTypes.number,
```